CONTRIBUTING.md

```diff
@@ -51,10 +51,9 @@ Committers for this project are documented in the project's [CODEOWNERS](CODEOWN
 
 - Make sure you follow the [review and commit guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines), in particular:
 
-    - Ensure that each commit is correct independently (i.e., each commit should compile and pass tests).
-      When possible, the size of the commit should be reduced for ease of review, while still ensuring
-      that it is independently correct.
-
+    - Ensure that each commit is correct independently. Each commit should compile and pass tests.
+    - Squash the commit so that it is no larger than the size required for it to work independently.
```

Comment:
IMHO somehow this does not capture the original message that the commits should be small and independent so that is easier to review.

Reply:
```suggestion
    - When possible, reduce the size of the commit for ease of review.
```

@ajaygeorge , good point! I agree that "no larger than the size required" doesn't state the why of "to make reviews easier". 

My new suggestion here attempts to restore that meaning. I also changed the sentence to active voice (reduce the size of the commit...) and dropped the independently correct part of the sentence as repetitive from the list item immediately before this. 

I hope this helps! Let me know what you think.

Reply:
SGTM!